GHzDACs/GHzAD_v7.md

```diff
@@ -0,0 +1,437 @@
+# Ethernet Commands for GHzADC
+
+John Martinis
+
+## General specifications
+
+FPGA interface for GHzADC.
+Prototype is version/build 1, 2 and 6.
+Revised are others.
+DO NOT MIX as FPGA pinouts changed.  
+
+The two AD converters on the board are each 8 bits at 1 Gs/s.
+To simplify computation, data from adjacent times are summed, giving a net data rate at 500 Ms/s, 9 bits.
+With two (I and Q) channels, this gives a system bandwidth of +- 250 MHz.  
+
+The Ethernet write commands are very similar to the GHzDAC board.
+The two basic command types are: 
+1. SRAM Write   (to download sine functions)
+2. Register Write (controls the board function)  
+
+Ethernet commands have Preamble, Destination, and Source fields, followed by two length bytes and the data bytes.
+The SRAM and register commands are recognized according to the value of the length bytes, which gives the total number of bytes of the data field d(1)..d(length).
+Ethernet packets are specified to have data-field lengths from 46 to 1500.  
+
+### MAC address
+
+MAC address of card is 0.1.202.170.1.(00sw), where sw[5..0] is the dipswitch value on the board with on=0.
+This address is from a now defunct company of Steve Waltman.
+The 1 value (before 00sw) indicates this is an AD board.
+
+### LEDs
+
+
+LED[0] light when either FPGA PLL or external 1 GHz PLL not locked/pgrmd  
+LED[1] light indicates bad clocking phase of either AD chip  
+LED[2] light indicates over range of I input  
+LED[3] light indicates over range of Q input  
+
+
+### Monitors
+
+mon[0] is defined by register, see page below  
+mon[1] is defined by register
+
+### Revision history
+
```

Comment:
I find it useful to have this because it tells me what to expect changed from revision to revision during debugging.

Moved to end.